go/vt/sqlparser/parse_test.go

```diff
@@ -1289,7 +1285,7 @@ var (
 		output: "use `ks:-80@master`",
 	}, {
 		input:  "describe foobar",
-		output: "otherread",
+		output: "show columns on foobar",
 	}, {
 		input:  "desc foobar",
```

Comment:
Something to come back to...we should probably support `desc`, since it looks like it's basically free.

Reply:
Yup